src/lib/backend.ai-client-esm.ts

```diff
@@ -2128,6 +2130,51 @@ class Agent {
   }
 }
 
+class AgentSummary {
+  public client: any;
+
+  /**
+   * Agent API wrapper.
+   *
+   * @param {Client} client - the Client API wrapper object to bind
+   */
+  constructor(client) {
+    this.client = client;
+  }
+
+  /**
+   * List of agent summary.
+   *
+   * @param {string} status - Status to query. Should be one of 'ALIVE', 'PREPARING', 'TERMINATING' and 'TERMINATED'.
+   * @param {array} fields - Fields to query. Queryable fields are:  id, status, scaling_group, schedulable, schedulable, available_slots, occupied_slots.
+   * @param {number} limit - limit number of query items.
+   * @param {number} offset - offset for item query. Useful for pagination.
+   * @param {number} timeout - timeout for the request. Default uses SDK default. (5 sec.) 
+   */
+  async list(status = 'ALIVE', fields = ["id", "status", "scaling_group", "schedulable", "available_slots", "occupied_slots", "architecture"],
```

Comment:
It would be great to treat status as a type since there are only 4 cases in status "ALIVE", "PREPARING", "TERMINATING", "TERMINATED".
It is a similar case that I mentioned in #1441 code review, so maybe we can add a single issue about that (since it's just about declaring type, right?).

Reply:
Yes, and there are several situations like this throughout the component, so it would be good to create a separate issue that does type alias.

Reply:
I'll handle it from https://github.com/lablup/backend.ai-webui/issues/1448!